boards/DIM/Inc/App/App_SevenSegDisplays.h

```diff
@@ -43,11 +46,24 @@ void App_SevenSegDisplays_Destroy(struct SevenSegDisplays *seven_seg_displays);
  * @param hex_digits The hexadecimal digits to set
  * @param num_hex_digits The number of hexadecimal digits to write, up to a
  *                       maximum of 3
- * @return ERROR_CODE_INVALID_ARGS if more than 3 hexadecimal digits are
+ * @return EXIT_CODE_INVALID_ARGS if more than 3 hexadecimal digits are
  *         requested, or if any of the requested digits contains a value that
  *         is not in the range of [0x0-0xF]
  */
-ErrorCode App_SevenSegDisplays_SetHexDigits(
+ExitCode App_SevenSegDisplays_SetHexDigits(
     const struct SevenSegDisplays *seven_seg_displays,
     const uint8_t                  hex_digits[],
     size_t                         num_hex_digits);
+
+/**
+ * Set an unsigned base-10 value on the given group of 7-segment displays
+ * @note The value that can be set is constrained by the number of 7-segment
+ *       displays. Use the return code to see if the requested value is valid.
+ * @param seven_seg_displays The group of 7-segment displays to set base-10
+ *                           value for
+ * @param value The unsigned base-10 value to set
```

Comment:
nit; might be to better to say "to display" then "to set", since that's the effect the user of this is likely aiming for.

Reply:
But then should the function name be "DisplayUnsignedBase10"? I do agree that "to set" is a bit vague and weak though

Reply:
Changed only "to set" to "to display" for now